src/components/Banner/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styles from './styles.module.css';
+
+const Banner: React.FC = () => (
+  <div className={styles.banner}>
+    <h1 className={styles.header}>Conduit</h1>
```

Comment:
As it's inherit element from `div.banner` could you please rename class `header` to `banner__header` according to BEM?